provisioning_templates/snippet/ssh_authorized_keys.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+kind: snippet
+name: ssh_authorized_keys
+model: ProvisioningTemplate
+snippet: true
+%>
+
+<%#
+SSH keys setup snippet
+
+Parameters:
+
+ssh_authorized_keys: public keys to be put in /root/.ssh/authorized_keys
+
+This template sets up SSH keys in any host so that as long as your public
+SSH key is in ssh_authorized_keys, you can SSH into a host.
+%>
+
+<% if !host_param('ssh_authorized_keys').blank? %>
+<% authorized_keys = host_param('ssh_authorized_keys') %>
+<% ssh_user = 'root' %>
```

Comment:
does ssh user has to be root?

Reply:
The user could be configurable. Sure. The remote_execution_ssh_keys adds sudo to the corresponding user if needed. Also i think there is a way to add the ssh_key of the foreman user into the newly created machine. But, i think, all of those serve different purposes.

I tried to mimic the logic of the `ssh_authorized_keys` parameter as used here:
https://github.com/theforeman/community-templates/blob/develop/provisioning_templates/provision/coreos_provision.erb#L32
And bring it to more platforms.